src/Compilers/VisualBasic/Portable/SymbolDisplay/CustomSymbolDisplayFormatter.vb

```diff
@@ -124,5 +120,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shared Function ShortNameWithTypeArgsAndContainingTypes(symbol As Symbol) As FormattedSymbol
             Return New FormattedSymbol(symbol, ShortWithTypeArgsAndContainingTypesFormat)
         End Function
+
+        Public Shared Function DefaultErrorFormat(symbol As Symbol) As FormattedSymbol
+            Return New FormattedSymbol(symbol, SymbolDisplayFormat.VisualBasicErrorMessageFormat)
+        End Function
+
+        Public Shared Function DefaultErrorFormatIfErrorSymbol(arg As Object) As Object
+
+            Dim symbol = TryCast(arg, Symbol)
```

Comment:
But it looks like we are interested in TypeSymbols only, why not check for TypeSymbol from the start?